qa/rpc-tests/signmessages.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+# Copyright (c) 2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+
+class SignRawTransactionsTest(BitcoinTestFramework):
+    """Tests transaction signing via RPC command "signrawtransaction"."""
+
+    def setup_chain(self):
+        print('Initializing test directory ' + self.options.tmpdir)
+        initialize_chain_clean(self.options.tmpdir, 1)
+
+    def setup_network(self, split=False):
+        self.nodes = start_nodes(1, self.options.tmpdir)
+        self.is_network_split = False
+
+    def run_test(self):
+        message = 'This is just a test message'
+
+        # Test the signing with a privkey
+        privKey = 'cUeKHd5orzT3mz8P9pxyREHfsWtVfgsfDjiZZBcjUBAaGk1BTj7N'
+        address = 'mpLQjfK79b7CCV4VMJWEWAj5Mpx8Up5zxB'
+        try:
+            signature = self.nodes[0].signmessagewithprivkey(privKey, message)
+        except JSONRPCException as e:
+            assert("Invalid private key" not in e.error['message'])
```

Comment:
Right, if you feel like adding a case that should fail to check if it fails properly, you are very welcome to do so.